### PR TITLE
fix nonbinary pronouns

### PR DESCRIPTION
**Pull request by cjquines**
agents with nonbinary gender should be referred to with the nonbinary pronoun set.

in typical english usage this is treated as a plural third-person pronoun ("they sing" rather than "they sings") so we add a method that treats the agent as plural.

didn't test this too hard oops

### CODE REVIEW

File: curveship.js [hunk 1]
```diff
@@ -598,9 +598,22 @@ class Narrator {
       return "future";
     }
   }
+  determineAgentNumber(ev, spin) {
+    let subject = world.ev[ev.tag].agent;
+    if (subject instanceof ExistentGroup) {
+      return 2;
+    }
+    let isPronoun = this.whatPronoun(subject, spin, "subject", ev);
```
Comment (cjquines) [inline curveship.js:606]: i believe so—this line checks if pronouns are being used for this event and subject

**Review by nickmontfort — CHANGES_REQUESTED** [review]
The Missing Names Test in text/nameless now fails.

**Comment by cjquines** [comment]
my bad—should be fixed. checked all the other tests and edited a test too.

**Comment by nickmontfort** [comment]
Looks good. I am going to add one line to text/nameless/story.js after approving this so that there is a regression test for your contribution.

**Pull request merged**